packages/components/releaseNotes/labkey/components.md

```diff
@@ -1,6 +1,10 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages.
 
+### version 0.33.0-fb-queryMetadataEditor-0.4
+*Released*: TBD
+* Item 6803: New QueryMetadata Editor
```

Comment:
Can you update this message to be more related to what changes are made here. The QueryMetdataEditor component isn't actually in this repo, this is just a set of changes to support that scenario right?